xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -1417,9 +1445,11 @@ bool CGUIMediaWindow::OnPlayAndQueueMedia(const CFileItemPtr &item, std::string
   return true;
 }
 
-// \brief Synchronize the fileitems with the playlistplayer
-// It recreated the playlist of the playlistplayer based
-// on the fileitems of the window
+/*!
+ * \brief Synchronize the fileitems with the playlistplayer
+ * It recreated the playlist of the playlistplayer based
```

Comment:
lol yeah that's what we used to draw on the blackboard to annoy the teacher back in the day .D